estate_account/models/estate_property.py

```diff
@@ -0,0 +1,25 @@
+from odoo import Command, models
+
+
+class EstateProperty(models.Model):
+    _inherit = "estate.property"
+
+    def estate_sold(self):
+        self.env["account.move"].create({
+            "name": "Test",
+            "move_type": "out_invoice",
+            "partner_id": self.env["estate.property"].buyer_id,
+            "line_ids": [
```

Comment:
```suggestion
            "invoice_line_ids": [
```
You are asked to use invoice_line_ids, not line_ids. Also, you forgot the line with the main amount, so far you have only the tax and admin fee